src/components/ColorBlock/color-block.scss

```diff
@@ -0,0 +1,17 @@
+.color-block {
+  position: relative;
+}
```

Comment:
```suggestion
  font-size: .875rem;
}
```

Reply:
I think this should actually be a smaller font size. :D

Reply:
cool, implemented!